Dockerfile

```diff
@@ -31,7 +32,7 @@ RUN apk add --no-cache libgcc pcre-dev
 # Fix for 'Error loading shared library libpcre.so.3: No such file or directory'
 RUN ln -s /usr/lib/libpcre.so /usr/lib/libpcre.so.3
 
-COPY --from=nim-build /app/build/wakunode /usr/bin/wakunode
+COPY --from=nim-build /app/build/wakunode2 /usr/bin/wakunode
```

Comment:
Shouldn't this hold the make target also?

Reply:
It might also be confusing to rename `wakunode2` to `wakunode`, since former is for v1. Maybe parameterize name so ENRYPOINT etc is `wakunode2`?

Reply:
Ah, I frogot to parametrize that.